pkg/util/configspec.go

```diff
@@ -190,3 +191,35 @@ func EnsureMinHardwareVersionInConfigSpec(
 	}
 	configSpec.Version = configSpecHwVersion.String()
 }
+
+// SafeConfigSpecToString returns the string-ified version of the provided
+// ConfigSpec, first trying to use the special JSON encoder, then defaulting to
+// the normal JSON encoder.
+func SafeConfigSpecToString(
```

Comment:
@dougm We should probably just add `String() string` to the ConfigSpec type in GoVmomi and implement this function there.

Reply:
Yeah, I was thinking the same after I posted the above.

Reply:
And lets make docs clear on the func it's not meant for JSON encoding, but rather when wanting to print the ConfigSpec as a string.